connect/runtime/src/test/java/org/apache/kafka/connect/integration/BlockingConnectorTest.java

```diff
@@ -350,13 +353,16 @@ private void assertRequestTimesOut(String requestDescription, ThrowingRunnable r
     }
 
     private static class Block {
```

Comment:
> can you make this public to allow OffsetsApiIntegrationTest to use the latch?

Yep, done 👍 

> and do you think that maybe these connectors should be moved out of this test to a common reusable class?

I do think this would be cleaner, but it'd fairly involved. Do you think it's alright to merge this as-is without blocking on that?

Reply:
> Do you think it's alright to merge this as-is without blocking on that?

Yep not a blocker.